.m365rc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We shouldn't include this file in the CLI repo. Let's remove it

Reply:
This file was deleted yesterday in the 'Delete .m365rc.json' commit. Do you still see it?

Reply:
It's gone now 👍